src/main/jbake/content/hawkular-services/docs/installation-guide/secure-comm.adoc

```diff
@@ -72,6 +72,13 @@ First, you must add a security-realm in standalone/configuration/standalone.xml:
         </ssl>
       </server-identities>
     </security-realm>
+    <security-realm name="HawkularAgentRealm">
+      <server-identities>
+        <ssl>
+          <truststore path="hawkular.keystore" relative-to="jboss.server.config.dir" keystore-password="hawkular" />
+        </ssl>
+      </server-identities>
+    </security-realm>
 ----
```

Comment:
This is what I've [added](https://github.com/hawkular/hawkular-services/pull/133/files#diff-e1e40939b471a17e473d9220afb7128bR41) and it worked

```xml
  <authentication>
    <truststore path="hawkular.keystore" relative-to="jboss.server.config.dir" keystore-password="hawkular"/>
  </authentication>
```

If the above code does the similar thing, then lgtm